gettsim/data/unterhalt.yaml

```diff
@@ -1,420 +1,51 @@
-  name:
-    de: Unterhaltsvorschuss für Kinder ab 13 Jahren. Anteil am Kinderfreibetrag.
-    en:
-  description:
-    de: §1612a (1) Nr.3 BGB
-    en:
-  values:
-    2008-01-01:
-      value: 1.17
-      note: Artikel 1 G. v. 21.12.2007 BGBl. I S. 2631
-uhv6_amount:
-  name:
-    de: Mindestunterhalt für Kinder bis 6 Jahre
-    en:
-  description:
-    de: Mindestunterhalt für Kinder in der ersten Altersstufe (bis zur Vollendung des sechsten Lebensjahrs, nach § 1612a BGB). § 1 Mindesunterhaltsverordnung.
-    en:
-  values:
-    2016-01-01:
-      value: 335
-      note: V. v. 03.12.2015 BGBl. I S. 2188
-    2017-01-01:
-      value: 342
-      note: V. v. 03.12.2015 BGBl. I S. 2188
-    2018-01-01:
-      value: 348
-      note: Artikel 1 V. v. 28.09.2017 BGBl. I S. 3525
-    2019-01-01:
-      value: 354
-      note: Artikel 1 V. v. 28.09.2017 BGBl. I S. 3525
-    2020-01-01:
-      value: 369
-      note: Artikel 1 V. v. 12.09.2019 BGBl. I S. 1393
-    2021-01-01:
-      value: 378
-      note: Artikel 1 V. v. 12.09.2019 BGBl. I S. 1393
-uhv12_amount:
-  name:
-    de: Mindestunterhalt für Kinder von 7-12 Jahren
-    en:
-  description:
-    de: Mindestunterhalt für Kinder in der zweiten Altersstufe (vom siebten bis zur Vollendung des zwölften Lebensjahrs, nach § 1612a BGB). § 1 Mindesunterhaltsverordnung.
-    en:
-  values:
-    2016-01-01:
-      value: 384
-      note: V. v. 03.12.2015 BGBl. I S. 2188
-    2017-01-01:
-      value: 393
-      note: V. v. 03.12.2015 BGBl. I S. 2188
-    2018-01-01:
-      value: 399
-      note: Artikel 1 V. v. 28.09.2017 BGBl. I S. 3525
-    2019-01-01:
-      value: 406
-      note: Artikel 1 V. v. 28.09.2017 BGBl. I S. 3525
-    2020-01-01:
-      value: 424
-      note: Artikel 1 V. v. 12.09.2019 BGBl. I S. 1393
-    2021-01-01:
-      value: 434
-      note: Artikel 1 V. v. 12.09.2019 BGBl. I S. 1393
-uhv17_amount:
-  name:
-    de: Mindestunterhalt für Kinder von 13-17 Jahren
-    en:
-  description:
-    de: Mindestunterhalt für Kinder in der dritten Altersstufe (vom 13. Lebensjahr an, solange minderjährig, nach § 1612a BGB). § 1 Mindesunterhaltsverordnung.
-    en:
-  values:
-    2016-01-01:
-      value: 450
-      note: V. v. 03.12.2015 BGBl. I S. 2188
-    2017-01-01:
-      value: 460
-      note: V. v. 03.12.2015 BGBl. I S. 2188
-    2018-01-01:
-      value: 467
-      note: Artikel 1 V. v. 28.09.2017 BGBl. I S. 3525
-    2019-01-01:
-      value: 476
-      note: Artikel 1 V. v. 28.09.2017 BGBl. I S. 3525
-    2020-01-01:
-      value: 497
-      note: Artikel 1 V. v. 12.09.2019 BGBl. I S. 1393
-    2021-01-01:
-      value: 508
-      note: Artikel 1 V. v. 12.09.2019 BGBl. I S. 1393
-uhv17_inc:
+mindestunterhalt:
+  name:
+    de: Mindestunterhalt für Kinder
+    en: Minimum Child Alimony
+  description:
+    de: § 1612a BGB, § 1 Mindesunterhaltsverordnung. Mindestunterhalt für Kinder in Abhängigkeit des Alters.
+    en:
+  type: euro
+  time: month
+  2016-01-01:
+    6: 335
```

Comment:
I would either add some more explanation: Saying kids under the age given or in between. Even better I think would be to give the keys not integer values and instead say: `unter_6`, `6_bis_12`etc.

Reply:
I went for the former (more concise keys)

Reply:
Okay!